iFixit/toolbox
# Add form components

**Part of https://github.com/iFixit/ifixit/issues/19628**

Organizes Toolbox repo and adds form components:
  - [x] `TextField`
  - [x] `RadioGroup`
  - [x] `Textarea`

> **CR note:** Don't worry about the `+10,584` lines added. Most of the added lines are from the generated `package-lock.json`.

# QA

* Go to https://deploy-preview-2--toolbox.netlify.com/
* Assert that all the components (listed above) exist
* Assert that all components work as expected across all browsers

cc @tvler
